packages/primitives/tooltip/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './src/tooltip-arrow.directive';
+export * from './src/tooltip-content-attributes.directive';
+export * from './src/tooltip-content.directive';
+export * from './src/tooltip-root.directive';
+export * from './src/tooltip-trigger.directive';
+export * from './src/tooltip.types';
```

Comment:
Tooltip implementation requires all the imports, please ensure you also export the NgModule. 

You can refer to how this is done in other primitives for consistency.

Reply:
done